ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -204,6 +206,22 @@ lazy_static::lazy_static! {
         "icu::properties::SentenceBreak",
         "icu::properties::WordBreak",
 
+        // Experimental
+        "icu::properties::maps::load_canonical_combining_class",
+
+        // Unnecessary for now
+        "icu::properties::maps::CodePointMapDataBorrowed::iter_ranges",
```

Comment:
Issue: These are not unnecessary. They are core functionality used by e.g. irregex.

Reply:
What's the plan for exposing them? I was under the impression their primary purpose was for constructing inversion lists and such manually.

Reply:
Or, more pertinently, are these in scope for 1.0?